### PR TITLE
Feat/autorenew renew actions

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

* Rename action for manual renew
* Disable renew for autorenewed active services
* Disable renew for services with debt 

Several tickets related as they impact the same files : 
 MANAGER-1802
MANAGER-1807
MANAGER-1806

### CODE REVIEW

File: client/app/account/billing/autoRenew/billing-autoRenew.controller.js [hunk 1]
```diff
@@ -343,7 +343,7 @@ angular.module('Billing.controllers').controller('Billing.controllers.AutoRenew'
             }
 
             if (!service.renew) {
-              _.set(service, 'renew', { automatic: false, force: false });
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/billing-autoRenew.controller.js]: 👍

File: client/app/account/billing/autoRenew/billing-autoRenew.constants.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 angular
   .module('Billing.constants')
-  .constant('SUBSIDIARIES_WITH_RECENT_AUTORENEW', ['FR', 'CA', 'QC', 'WE', 'WS', 'ASIA', 'SG', 'AU']);
+  .constant('SUBSIDIARIES_WITH_RECENT_AUTORENEW', ['FR', 'CA', 'QC', 'WE', 'WS', 'ASIA', 'SG', 'AU'])
+  .constant('DEBT_STATUS', ['PENDING_DEBT', 'UN_PAID']);
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/billing-autoRenew.constants.js]: What's the difference ?

File: client/app/account/billing/autoRenew/billing-autoRenew.controller.js [hunk 3]
```diff
@@ -693,6 +693,10 @@ angular.module('Billing.controllers').controller('Billing.controllers.AutoRenew'
       });
     }
 
+    function isInDebt(service) {
+      return _.includes(DEBT_STATUS, service.status);
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/billing-autoRenew.controller.js]:
```suggestion
      return DEBT_STATUS.includes(service.status);
```

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 4]
```diff
@@ -33,7 +33,8 @@
    <translation id="autorenew_service_renew_frequency_value" qtlid="402932">Tous les {0} mois</translation>
    <translation id="autorenew_service_renew_years_frequency_value" qtlid="402945">Automatique : tous les {0} ans</translation>
    <translation id="autorenew_service_renew_paid" qtlid="37515">Payé</translation>
-   <translation id="manualrenew_service_update_button_caption" qtlid="416111">Payer le service</translation>
+   <translation id="manualrenew_service_renew_button">Renouveler le service</translation>
+   <translation id="manualrenew_service_renew_button_title" qtlid="416111">Renouveler le service : {0} (Nouvelle fenêtre)</translation>
```
Comment (JeremyDec) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: `qtlid` ! :)

**Pull request merged**